test/utils/image/manifest.go

```diff
@@ -30,17 +30,18 @@ import (
 
 // RegistryList holds public and private image registries
 type RegistryList struct {
-	GcAuthenticatedRegistry  string `yaml:"gcAuthenticatedRegistry"`
-	PromoterE2eRegistry      string `yaml:"promoterE2eRegistry"`
-	BuildImageRegistry       string `yaml:"buildImageRegistry"`
-	InvalidRegistry          string `yaml:"invalidRegistry"`
-	GcEtcdRegistry           string `yaml:"gcEtcdRegistry"`
-	GcRegistry               string `yaml:"gcRegistry"`
-	SigStorageRegistry       string `yaml:"sigStorageRegistry"`
-	PrivateRegistry          string `yaml:"privateRegistry"`
-	MicrosoftRegistry        string `yaml:"microsoftRegistry"`
-	DockerLibraryRegistry    string `yaml:"dockerLibraryRegistry"`
-	CloudProviderGcpRegistry string `yaml:"cloudProviderGcpRegistry"`
+	GcAuthenticatedRegistry   string `yaml:"gcAuthenticatedRegistry"`
+	PromoterE2eRegistry       string `yaml:"promoterE2eRegistry"`
+	BuildImageRegistry        string `yaml:"buildImageRegistry"`
+	InvalidRegistry           string `yaml:"invalidRegistry"`
+	GcEtcdRegistry            string `yaml:"gcEtcdRegistry"`
+	GcRegistry                string `yaml:"gcRegistry"`
+	SigStorageRegistry        string `yaml:"sigStorageRegistry"`
+	SigStorageStagingRegistry string `yaml:"sigStorageStagingRegistry"`
```

Comment:
This looks odd. What manifest needs an image from a staging registry? I can see `csi-snapshotter` below, but I can't find anything in today's master. 

Maybe add a comment that this registry is used temporarily for pre-release images.